### PR TITLE
docs(graphql): rewrote search.md

### LINKED ISSUE #952: GraphQL README.md: give examples of how to use pagination in search and/or other calls
https://github.com/laurentiuNiculae/zot/pull/66

### PR DESCRIPTION
**Pull request by andaaron**
Signed-off-by: Catalin Hofnar <catalin.hofnar@gmail.com>
(cherry picked from commit 2268fa0510b32b27f2c1f71e9889ec769877553b)
Signed-off-by: Andrei Aaron <aaaron@luxoft.com>

Resolves #952 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by rchincha** [comment]
Pls don't move search.md into README.md, there are OCI extension implications. Just append to the existing search.md pls.

File: pkg/extensions/search/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+search.md
```
Comment (rchincha) [inline pkg/extensions/search/README.md:1]: Add a little more text here

`
# `search` extension

Details about the APIs are specified [here](./search.md)

`

Or even remove this README.md entirely, this is a new file which was added right?

File: pkg/extensions/search/search.md [hunk 2]
```diff
@@ -1,186 +1,673 @@
-# `search` 
+# `search`
 
 `search` component provides efficient and enhanced registry search capabilities using graphQL backend.
-	
+
 | Supported queries | Input | Ouput | Description | graphQL query |
 | --- | --- | --- | --- | --- |
-| [Search images by digest](#search-images-by-digest) | digest | image list | Search all repositories in the registry and return list of images that matches given digest (manifest, config or layers) | ImageListForDigest |
```
Comment (rchincha) [inline pkg/extensions/search/search.md:7]: Why change the position of these? Would ideally either amend-in-place or append new entries

File: pkg/extensions/search/README.md (hunk 1 above)
Comment (andaaron) [reply pkg/extensions/search/README.md:1]: It's a symlink.

File: pkg/extensions/search/search.md (hunk 2 above)
Comment (andaaron) [reply pkg/extensions/search/search.md:7]: Will update the doc. No reason that I am aware of.

**Review by rchincha — APPROVED** [review]
lgtm

**Pull request merged**